### DISCUSSION TITLE
App Specific Customization? [Moved to Issue #279]

### DISCUSSION THREAD

**Discussion by evkaw** [noah-nuebling/mac-mouse-fix#186] [Other]
I would love to see app-specific actions since as of now as I know, there are only like 3 things to change as it is app specific.

**Comment by noah-nuebling** [comment]
Hey Evka! I'd love to add fully customizable app-specific settings, and have a pretty good idea for how I wanna do it. But it's a long way off at this point.

**Comment by evkaw** [comment]
sure, take your time! love the app

**Comment by noah-nuebling** [comment]
Hey! Moving this topic to #279. Please use that issue for further discussion of App-Specific customizations, and let me know if you have more questions. Thanks!
